third_party/docfx/templates/devsite/ManagedReference.common.js

```diff
@@ -44,6 +44,9 @@ exports.transform = function (model) {
     }
   }
 
+  if (model.summary != null && model.friendlyApiName) {

```

Comment:
I need to constantly remind myself that `null` means the YAML entry is there but value is empty, whereas the field not existing is a completely different check (or I got it mixed up but either or). I wonder if it's better to check with a better named function, such as
```
if (!isValueEmpty(model.summary) && model.friendlyApiName) {
```
just a thought. If this is also expected in JS and my lack expertise is the cause then let's leave it as is.

Reply:
Agreed it's not so obvious -- I added a clarifying comment.